fluent-bit/Dockerfile

```diff
@@ -65,7 +65,7 @@ FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4
 ARG UPSTREAM_VERSION
 ARG RELEASE
 
-ENV SUMMARY="UBI based Fluent Bit" \
+ENV SUMMARY="Fluent Bit is a fast Log Processor and Forwarder" \
     DESCRIPTION="Fluent Bit is a fast Log Processor and Forwarder"
```

Comment:
Here Summary and Description are the same, while for example `kube-rbac-proxy` keeps them distinct. Again, please stick to one convention or explain why it's not suitable.